src/traversals/greedy_color.jl

```diff
@@ -39,28 +39,27 @@ function perm_greedy_color(g::AbstractGraph, seq::Vector{T}) where T <: Integer
         end
     end
 
-    return coloring{T}(maximum(cols), cols)
+    return Coloring{T}(maximum(cols), cols)
 end
 
 """
     degree_greedy_color(g)
 
 Color graph `g` iteratively in the descending order of the degree of the vertices.
 """
-function degree_greedy_color(g::AbstractGraph{T}) where T <: Integer 
+function degree_greedy_color(g::AbstractGraph{T}) where {T <: Integer} 
     seq = convert(Vector{T}, sortperm(degree(g), rev=true)) 
     return perm_greedy_color(g, seq)
 end
 
 
 """
-    random_greedy_color(g, reps=1)
+    random_greedy_color(g, reps)
 
-Color graph `g` iteratively in a random order using a greedy heruistic
-and choose the best coloring out of `reps` such random coloring.
+Colors the graph `g` iteratively in a random order using a greedy heuristic
```

Comment:
We use action verbs to start docstrings. `Color` is correct here.

Reply:
Fixed!